MapleServer2/Packets/WalletPacket.cs

```diff
@@ -0,0 +1,42 @@
+﻿using MaplePacketLib2.Tools;
+using MapleServer2.Constants;
+using MapleServer2.Enums;
+
+namespace MapleServer2.Packets
+{
+    class WalletPacket
+    {
+        public static Packet UpdateWallet(CurrencyType type, long amount)
+        {
+            byte currency = 0;
+            switch (type)
+            {
+                case CurrencyType.ValorToken:
+                    currency = 3;
```

Comment:
do we want to add currency type to the enum?
https://www.tutorialsteacher.com/csharp/csharp-enum

Reply:
Sure!